frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-card.scss

```diff
@@ -0,0 +1,52 @@
+$skeleton-animation: loading-skeleton 1s ease-out .15s infinite alternate;
```

Comment:
you should import these from `console-shared/styles/skeleton-screen`

Reply:
ack